.github/renovate.json5

```diff
@@ -0,0 +1,122 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  // We default to using the normal recommended config. Everything else is layered on top of this.
+  extends: [
+    'config:recommended',
+    // https://docs.renovatebot.com/presets-default/#automergepatch - Automerge lockfile maintenance, path, pin and separate minor and patch
+    ':automergePatch',
+    // Pin docker
+    'docker:pinDigests',
+    // Pin Github Actions
+    'helpers:pinGitHubActionDigests',
+    // Pin dev dependencies (See https://docs.renovatebot.com/upgrade-best-practices/#extends-pindevdependencies why this is a good idea for us)
+    ':pinDevDependencies',
+    // https://docs.renovatebot.com/presets-schedule/#scheduleweekly - Run renovate on early mondays instead of running it too often
+    'schedule:weekly',
+    // https://docs.renovatebot.com/presets-schedule/#scheduleautomergeweekdays - Run automerge on a weekdays (so it doesnt break on the weekend)
+    'schedule:automergeWeekdays',
+  ],
+  dependencyDashboard: true,
+  // https://docs.renovatebot.com/configuration-options/#osvvulnerabilityalerts - we use osv.dev here to not rely on Github
+  osvVulnerabilityAlerts: true,
+  // https://docs.renovatebot.com/configuration-options/#automergetype - We use branch to not get raced by commits to main
+  automergeType: 'branch',
+  npm: {
+    // https://docs.renovatebot.com/presets-npm/#npmunpublishsafe - This effectively waits 14 days before making a PR. This is somewhat of a safety measure and recommended by renovate at https://docs.renovatebot.com/upgrade-best-practices/#wait-two-weeks-before-automerging-third-party-dependencies
+    minimumReleaseAge: '14 days',
+  },
+  // https://docs.renovatebot.com/configuration-options/#lockfilemaintenance - Ensure the lockfile is maintained properly
+  lockFileMaintenance: {
+    enabled: true,
+  },
+  // Set the timezone for schedules
+  timezone: 'Europe/Berlin',
+  // Ensures that appVersion in the helm chart is tracked/updated too
+  customManagers: [
+    {
+      customType: 'regex',
+      datasourceTemplate: 'docker',
+      managerFilePatterns: ['/(^|/)Chart\\.yaml$/'],
+      matchStrings: [
+        '#\\s?renovate: image=(?<depName>.*?)\\s?appVersion:\\s?("|\')?(?<currentValue>[\\w+\\.\\-]*)("|\')',
+      ],
+    },
+  ],
+  // https://docs.renovatebot.com/configuration-options/#packagerules - Note that later overrides earlier rules
+  packageRules: [
+    // Do not automerge matrix-widget-api updates as they dont follow semver
```

Comment:
consider using `description` instead of comments for packageRules? https://docs.renovatebot.com/configuration-options/#description